cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js

```diff
@@ -204,6 +202,7 @@ describe('TLR: Item renew', () => {
     cy.wrap(instanceData.itemIds).each((item) => {
       cy.deleteItemViaApi(item);
     });
```

Comment:
Let's have these two tests related to TLR in one file with one describe.

Reply:
done